.github/workflows/filter_testsuite.yml

```diff
@@ -0,0 +1,105 @@
+name: Filter Testsuite
+
+on:
+  issue_comment:
+      types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        id: check
+        with:
+         result-encoding: string
+         script: |
+           const userName = context.payload.comment.user.login
+           if(userName == 'maxGimeno') {
+             const body = context.payload.comment.body
+             if(body.includes("/testme")) {             
+               return 'OK'
+             }
+           }
+           return 'stop'
+      - uses: actions/github-script@v3
+        if: steps.check.result != 'stop'
+        id: get_label
+        with:
+         result-encoding: string
+         script: |
+           //get branch name and username
+           const pr_url = context.payload.issue.pull_request.url
+           const pr_content = await github.request(pr_url)
+           const label = pr_content.data.head.label
+           console.log(label)
+           return label
```

Comment:
This step cannot be skipped, because a `comment` even does not have access to the full PR payload, and we need to request it using `github.request`.